lib/signaling/v2/twilioconnectiontransport.js

```diff
@@ -428,7 +489,7 @@ function setupEventListeners(transport) {
         return;
       }
       case 'disconnected':
-        twilioConnection.removeListener('message', handleMessage);
+        transport._twilioConnection.removeListener('message', handleMessage);
```

Comment:
What about the TwilioConnection instances that were replaced due to reconnection? Where do we remove those "message" handlers?

Reply:
Should we factor this out and let both this line and `createOrResetTwilioConnection` call this?